src/corelib/corelib.nuspec

```diff
@@ -32,11 +32,11 @@
     <!-- Runtime libraries -->
     <file src="bin\v3.5\$Configuration$\openstacknet.dll" target="lib\net35"/>
     <file src="bin\v3.5\$Configuration$\openstacknet.pdb" target="lib\net35"/>
-    <file src="..\..\docs\Api\v3.5\openstacknet.xml" target="lib\net35"/>
+    <file src="bin\v3.5\$Configuration$\openstacknet.xml" target="lib\net35"/>
```

Comment:
Thanks, I wasn't aware that we were relying that deeply on Sandcastle with respect to the inline doc and assumed it was just for the openstack.net/docs site.